auth_conversion.py

```diff
@@ -0,0 +1,110 @@
+import json
+import sqlite3
+from pathlib import Path
+from uuid import NAMESPACE_URL
+from uuid import uuid5
+
+ASREVIEW_PATH = Path("/Users/casperkaandorp/.asreview")
+DATABASE_NAME = "asreview.development.sqlite"
```

Comment:
Probably when people are actually using this, it should be the production version. Then also the debug parameter in the config should be false.